### ISSUE TITLE
Typst syntax scope was changed, so syntax highlighting broke

### ISSUE THREAD

**Issue by istudyatuni** [SublimeText/AFileIcon#72]
I cannot use [syntax highlighting for typst](https://github.com/hyrious/typst-syntax-highlight) because A File Icon replaces it with syntax definition in `~/.config/sublime-text/Packages/zzz A File Icon zzz/aliases/Typst.sublime-syntax`, but this syntax is empty

#### Expected Behavior

Syntax highlighting works (loading from Typst extension)

#### Current Behavior

Syntax loaded from `aliases`, and I should manually select `Typst` syntax highlighting

#### Possible Solution

Change `source.typst` to `text.typst` in `preferences/file_type_typst.tmPreferences` and `icons/icons.json`

#### Steps to Reproduce (for bugs)

_Tested in safe mode_

1. Install `Typst`
2. Open any `.typ` file - syntax highlighting works
3. Install `A File Icon`
4. Open any `.typ` file - syntax highlighting not works

#### Context

I just want to have working syntax highlighting

#### Your Environment

Arch linux, ST stable build 4143

<details><summary>Details</summary>
<p>

```
startup, version: 4143 linux x64 channel: stable
SAFE MODE - overriding packages, state and cache paths
executable: /opt/sublime_text/sublime_text
application: /opt/sublime_text
working dir: /home/[redacted]/.config/sublime-text/Packages/zzz A File Icon zzz/aliases
packages path: /home/[redacted]/.config/sublime-text-safe-mode/Packages
state path: /home/[redacted]/.config/sublime-text-safe-mode/Local
zip path: /opt/sublime_text/Packages
zip path: /home/[redacted]/.config/sublime-text-safe-mode/Installed Packages
ignored_packages: ["Vintage"]
generating syntax summary
generating meta info summary
pre session restore time: 1.73338
startup time: 1.80827
first paint time: 1.8667
reloading plugin Default.arithmetic
reloading plugin Default.auto_indent_tag
reloading plugin Default.block
reloading plugin Default.colors
reloading plugin Default.comment
reloading plugin Default.convert_color_scheme
reloading plugin Default.convert_syntax
reloading plugin Default.copy_path
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.html_print
reloading plugin Default.indentation
reloading plugin Default.install_package_control
reloading plugin Default.keymap
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.profile
reloading plugin Default.quick_panel
reloading plugin Default.rename
reloading plugin Default.run_syntax_tests
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.settings
reloading plugin Default.show_scope_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.ui
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin ShellScript.ShellScript
plugins loaded
reloading python 3.3 plugin Package Control.1_reloader
reloading python 3.3 plugin Package Control.2_bootstrap
reloading python 3.3 plugin Package Control.Package Control
Package Control: Not running package cleanup since bootstrapping is not yet complete
reloading python 3.3 plugin 0_package_control_loader.00-package_control
reloading settings Packages/User/Package Control.sublime-settings
ignored packages updated to: ["Typst", "Vintage"]
reloading settings Packages/User/Package Control.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings
reloading settings Packages/User/Package Control.sublime-settings
ignored packages updated to: ["Vintage"]
reloading settings Packages/User/Package Control.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings
generating meta info summary
generating syntax summary
reloading settings Packages/User/Preferences.sublime-settings
ignored packages updated to: ["A File Icon", "Vintage"]
reloading settings Packages/User/Package Control.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings
reloading settings Packages/User/Package Control.sublime-settings
ignored packages updated to: ["Vintage"]
reloading plugin A File Icon.plugin
reloading settings Packages/User/Package Control.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings
generating meta info summary
generating syntax summary
reloading settings Packages/User/Preferences.sublime-settings
Unable to open /home/[redacted]/.config/sublime-text-safe-mode/Packages/A File Icon/.sublime/A File Icon.sublime-settings
Unable to open /home/[redacted]/.config/sublime-text-safe-mode/Packages/User/A File Icon.sublime-settings
reloading settings Packages/User/A File Icon.sublime-settings
```

</p>
</details>

**Comment by hyrious** [comment]
😅 That must because of me changing that to `text.typst` 1 month ago (https://github.com/hyrious/typst-syntax-highlight/issues/3) _before_ my plugin getting into the package control, where I used `source.typst` before that.

**Comment by deathaxe** [comment]
Fixed by v3.24.1
